nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/hosted/ConanHostedMetadataFacetSupportTest.java

```diff
@@ -63,6 +64,22 @@ public void generateAssetPackagesDownloadUrls() throws Exception {
     assertThat(actual, is(expected));
   }
 
+  @Test
+  public void generateAssetPackagesDownloadUrlsHashNotFound() throws Exception {
+    String assetPath =
```

Comment:
use `Paths.get(dir, dir, ...)` construction

Reply:
all tests use this approach. I would like to use one approach for all tests